packages/sdk-react/src/components/payables/PayablesTable/PayablesTable.tsx

```diff
@@ -504,7 +504,16 @@ const PayablesTableBase = ({
 const usePayablesTableSummaryData = () => {
   const { api, queryClient } = useMoniteContext();
 
-  api.payables.getPayablesAnalytics.invalidateQueries(queryClient);
+  if (queryClient) {
+    api.payables.getPayablesAnalytics.invalidateQueries(queryClient);
+  }
 
-  return api.payables.getPayablesAnalytics.useQuery();
+  return queryClient
+    ? api.payables.getPayablesAnalytics.useQuery(
+        {},
+        {
+          enabled: true,
+        }
+      )
+    : { data: null, isLoading: true };
```

Comment:
Did you intend to set the loading state to true when the `queryClient` isn't set?

Reply:
If queryClient is not ready, we return a default state { data: null, isLoading: true } instead of calling the query